src/reference/asciidoc/kafka.adoc

```diff
@@ -45,7 +47,7 @@ interface ProducerCallback<K, V, T> {
 
 ----
 
-The first 3 methods require that a default topic has been provided to the template.
+The sendDefault api's require that a default topic has been provided to the template.
```

Comment:
I also would like to see new headers for the `timestamp` in the paragraph about `ListenerContainer`, more precisely in the sample code for the `@KafkaListener(id = "qux", topicPattern = "myTopic1")`